code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -348,22 +350,9 @@
 	camera = null
 	light_overlay = "helm_light"
 	desc = "A bubble helmet that maximizes the field of view. A state of the art holographic display provides a stream of information"
+	icon = 'icons/clothing/rigs/helmets/helmet_null.dmi'
 
-//All in one suit
 /obj/item/clothing/suit/space/rig/zero
 	breach_threshold = 18
 	body_parts_covered = SLOT_UPPER_BODY|SLOT_LOWER_BODY|SLOT_LEGS|SLOT_FEET|SLOT_ARMS|SLOT_HANDS
-
-/obj/item/rig/zero/on_update_icon(var/update_mob_icon)
```

Comment:
are female versions non viable?
this is probably the only clothing item in game with actually different states.

Reply:
Oh I didn't even grok this as I deleted it. I can readd them if you want.

Reply:
nvm mixed that up with nullsuit's undersuit.